applications/dashboard/src/scripts/Authenticate/components/SSOMethods.tsx

```diff
@@ -0,0 +1,80 @@
+import { t } from '@core/application';
+import React from 'react';
+import Paragraph from '../../Forms/Paragraph';
+import Or from '../../Forms/Or';
+import {uniqueID, IComponentID} from '@core/Interfaces/componentIDs';
+
+interface IProps extends IComponentID {
+    includeOr?: boolean;
+    ssoMethods?: any[];
+}
+
+interface IState {
+    editable: boolean;
+    passwordAuthenticators?: any[];
+}
+
+export default class SSOMethods extends React.Component<IProps, IState> {
+    public ssoMethods: any[];
+    public includeOr: boolean;
+    public ID: string;
+
+    constructor(props) {
+        super(props);
+        this.ID = uniqueID(props, 'SSOMethods', true);
+        this.includeOr = props.includeOr || true;
+
+        this.state = {
+            editable: false,
+            passwordAuthenticators: props.passwordAuthenticators,
+        };
+
+    }
+
+    public handleClick (method):any {
```

Comment:
What's with the function signature here?  In this case you don't seem to be  using the parameter, but the signature for a React click handler is normally this

`functionName(event?: React.MouseEvent): v`